go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/circonus-labs/circonus-irondb-datasource
 
-go 1.15
+go 1.16
```

Comment:
.16?  Not .17?

Reply:
For some reason 1.17 in go.mod breaks mage :(